src/rpc/handlers/LedgerEntry.cpp

```diff
@@ -37,7 +37,13 @@ doLedgerEntry(Context const& context)
         if (!key.parseHex(request.at(JS(index)).as_string().c_str()))
             return Status{RippledError::rpcINVALID_PARAMS, "malformedIndex"};
     }
-    else if (request.contains(JS(account_root)))
+    if (request.contains(JS(ledger_index)))
+    {
+        if (!request.at(JS(ledger_index)).is_string() ||
```

Comment:
This is always true - ledger_index can't be both `string` and `int64`. 
Did you run this and passed it `ledger_index` of different types to see what happens?